tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -177,6 +181,154 @@ TEST( OutputReportTabularTest, GetUnitConversion )
 
 }
 
+TEST( OutputReportTabularTest, AllocateLoadComponentArraysTest )
+{
+	ShowMessage( "Begin Test: OutputReportTabularTest, AllocateLoadComponentArraysTest" );
+
+	AllocateLoadComponentArraysDoAllocate = true;
```

Comment:
@JasonGlazer Fewer global variables and closer to OO, consistent unit testing based around EnergyPlusFixture, 'hiding' unit testing specific details.

I say that last point because i'm sure `AllocateLoadComponentArraysDoAllocate` was a function static variable before someone put it in the anonymous namespace. This variable only needs to be reset for unit testing. During a normal simulation run it does not need to be accessed (and should not be accessed) by any other function and especially not any other implementation file. So it is a unit testing specific change and we should try to minimize the potential for change elsewhere in the code.

The whole point of the `clear_state()` function, which is automatically called by EnergyPlusFixture on `SetUp()` and `TearDown()`, is to take care of resetting global state so each unit tests starts fresh.